README.md

```diff
@@ -1,11 +1,35 @@
+# Libheap on ubuntu16.04
+
+This version is  easy to install  for ubuntu16.04.
+
+```
+apt-get install libc6-dbg:i386
+apt-get install libc6-dbg
+
+mkdir ~gdb
+cd ~/gdb
+git clone https://github.com/thinkycx/libheap
+pip install --user ./libheap/
+
+echo "python import sys" ~/.gdbinit
+echo "python sys.path.append('~/.local/lib/python2.7/site-packages')" ~/.gdbinit
+echo "python from libheap import *" >> ~/.gdbinit
+```
+
+Forked from https://github.com/cloudburst/libheap
+
+However, I changed the  `libheap/libheap/pydbg/pygdbpython.py` in line 137 to `except RuntimeError:` and annotate the print_error function as I think its doesn't make  work.Also, I changed the glibc vesion to `2.23`.
+
+
+
```

Comment:
Thank you for fixing the repo!  Would you mind removing the README.md changes from this pull request?  I am happy to merge the 63aaf8081b485149030ce278d41ebe476ea35245 commit, but I'd like to keep install instructions in the docs/ folder.